ts/features/it-wallet/saga/itwCredentialsSaga.ts

```diff
@@ -86,3 +95,39 @@ export function* handleCredentialsChecks(
     );
   }
 }
+
+/**
+ * This saga handles the request to add a credential to the wallet.
+ * NOTE: Currently it just returns true as we are only handling a mocked credential.
+ * @param action the request dispatched action with a credential as payload.
+ */
+export function* handleCredentialsAddCredential(
+  action: ActionType<typeof itwCredentialsAddCredential.request>
+): SagaIterator {
+  yield* put(
+    identificationRequest(false, true, undefined, {
+      label: I18n.t("global.buttons.cancel"),
+      onCancel: () =>
+        NavigationService.dispatchNavigationAction(CommonActions.goBack())
+    })
+  );
+
+  const res = yield* take(identificationSuccess);
+
+  if (isActionOf(identificationSuccess, res)) {
+    yield* put(itwCredentialsAddCredential.success(action.payload));
+    yield* put(itwLifecycleValid());
```

Comment:
Do we need to dispatch this action?

Reply:
Good catch thx, it is a copy/paste error.